### ISSUE TITLE
"p" is prepending to some summaries

### ISSUE THREAD

**Issue by greenlion** [oursqlcommunity-org/planet#39]
For example, my latest blog post (https://swanhart.livejournal.com/143173.html) shows:
pEngine condition pushdown changed in MySQL 8.0.20

The latest from severalnines (https://severalnines.com/database-blog/introduction-maxscale-administration-using-maxctrl-mariadb-cluster) shows:
pMariaDB Cluster consists of MariaDB Server

**Comment by jfg956** [comment]
Where (which url) do you see these "p" ?  A screenshot would help... Thanks.

**Comment by greenlion** [comment]
https://www.imgpaste.net/image/5SHU7

**Comment by jfg956** [comment]
Right, it is a bug in the html stripping. I cannot fix this now as I am away from my computer (I am writing this on my phone). Do not expect this to be fixed quickly as I might be away until end of August.

**Comment by dveeden** [comment]
![image](https://github.com/oursqlcommunity-org/planet/assets/1272980/2654e3a6-a8c4-4ab4-867d-8040843fc166)

Looks like I pasted html into blogger for this item that was standalone html instead of just the article part. So it has more html that confuses the html stripping.

I don't think the code should handle messy html like this, but it would be nice if it could.

Where is the HTML stripping code? maybe I can have a look at it to at least strip out `<p>` as that seems to happen often.

**Comment by dveeden** [comment]
Note that `/p` (for `</p>`) also sometimes shows up:
![image](https://github.com/oursqlcommunity-org/planet/assets/1272980/2b0cb493-4ea7-4ea7-b859-8bbf146267ad)

**Comment by jfg956** [comment]
Thanks for your interest in Planet for the MySQL Community Daniël.  And I see your post did not reach Planet MySQL, unclear if it is because it mentioned MariaDB or because you are an employee of an Oracle competitor.  Anyhow, it shows this Community Aggregator is needed.

HTLM stripping is done there:
- https://github.com/oursqlcommunity-org/planet/blob/master/template/index.html.erb#L119
- https://github.com/oursqlcommunity-org/planet/blob/master/template/atom.xml.erb#L24
- https://github.com/oursqlcommunity-org/planet/blob/master/template/rss.xml.erb#L28

I have not spent much time looking into this, because I believe the article snippet is not used by many (I myself mostly check the title to choose if I will go check a post or not).  But if you would like to try fixing this, the aggregator would look better.  I will send you a DM on MySQL Community Slack about this.  Cheers, J-F
